src/components/WishItem.tsx

```diff
@@ -79,9 +80,12 @@ const RemoveButton = styled.button`
     position: absolute;
 `;
 
-const WishItem = ({ amiiboWish }: WishProps) => {
-    const removeItem = async (wish: Amiibo) => {
-        console.log('prev wishlist: ' + JSON.stringify(wish));
+const WishItem = ({ amiiboWish, onRemove }: WishProps) => {
+    // const removeItem = async (wish: Amiibo) => {
+    //     console.log('prev wishlist: ' + JSON.stringify(wish));
+    // };
+    const handleAddToWishlist = () => {
```

Comment:
Just for cohesiveness, I think the name `handleRemoveFromWishlist` might be better, but I can fix it the next time I work on this feature since it's contained within this component!